doc/rst/usingchapel/prereqs.rst

```diff
@@ -44,6 +44,8 @@ for using Chapel:
     use the bundled LLVM or disable LLVM support (see
     :ref:`readme-chplenv.CHPL_LLVM`).
 
+     * Using the LLVM backend with LLVM 16 or newer requires C++17 support.
```

Comment:
This is good for now, but my own opinion is that we should just track LLVM's requirement here for our own compiler source code (i.e. require C++17 now because we bundle LLVM 17).